soapy/confParse.py

```diff
@@ -1028,7 +1034,11 @@ def calcParams(self):
         # Set some parameters to correct type
         self.position = numpy.array(self.position)
         self.wavelength = float(self.wavelength)
-
+        if self.pxlScale is not None:
+            logger.debug("Overriding sci FOV with pxlscale")
```

Comment:
I don't like that this happens silently and is only logged. It should check the pixel scale, no. of pixels and FOV (if all given) and raise a warning to let the user know that it is overwriting some of the wavefront sensor parameters if they don't all agree.